x-pack/plugins/security_solution/public/overview/components/recent_timelines/index.tsx

```diff
@@ -67,23 +68,28 @@ const StatefulRecentTimelinesComponent = React.memo<Props>(
     );
 
     const { fetchAllTimeline, timelines, loading } = useGetAllTimeline();
-
-    useEffect(() => {
-      fetchAllTimeline({
-        pageInfo: {
-          pageIndex: 1,
-          pageSize: PAGE_SIZE,
-        },
-        search: '',
-        sort: {
-          sortField: SortFieldTimeline.updated,
-          sortOrder: Direction.desc,
-        },
-        onlyUserFavorite: filterBy === 'favorites',
-        timelineType: TimelineType.default,
-      });
-      // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [filterBy]);
+    const timelineType = TimelineType.default;
+    const { templateTimelineType, timelineStatus } = useTimelineStatus({ timelineType });
+    useEffect(
+      () => {
+        fetchAllTimeline({
+          pageInfo: {
+            pageIndex: 1,
+            pageSize: PAGE_SIZE,
+          },
+          search: '',
+          sort: {
+            sortField: SortFieldTimeline.updated,
+            sortOrder: Direction.desc,
+          },
+          onlyUserFavorite: filterBy === 'favorites',
+          status: timelineStatus,
+          timelineType,
+          templateTimelineType,
+        });
+      }, // eslint-disable-next-line react-hooks/exhaustive-deps
+      [filterBy, timelineStatus, timelineType, templateTimelineType]
```

Comment:
Why fetchAllTimeline is not in the dependency?

Reply:
I have a problem here to add fetchAllTimeline as part of dependency. Once I add it, it hangs the app. Therefor currently I haven't added it as dependency that it should have been.